packages/tradershub/src/components/Base/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+type TProps = {
+    alignment?: 'bottom' | 'left' | 'right' | 'top';
+    className?: string;
+    isVisible: boolean;
+    message: string;
+};
+
+const Tooltip: React.FC<React.PropsWithChildren<TProps>> = ({
```

Comment:
At this point having `PropsWithChildren` abit redundant
Also important everything from React ontop
```suggestion
const Tooltip: FC<TProps> = ({
```

Reply:
Same here 
TSDocs description